app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,18 @@ def show
     @user = User.find(params[:user_id])
     @post = @user.posts.includes(:comments).find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
+    @post.author = current_user
```

Comment:
- [Optional] Dear @agneta1991, Great job on using the strong parameters for accepting the post_params, but a better way to implement this functionality will be to create a separate method to filter params and re-use it on different locations which you might need in the future. This will make your code `Single responsible` and `DRY/Dont repeat yourself`. [More about storing parameters here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).